### PR TITLE
Cata beta (draft)

### PR DESCRIPTION
**Pull request by Hendo72**

### CODE REVIEW

**Comment by Hendo72** [comment]
Can we merge this as a separate branch until we are ready to merge at launch?

**Review by Hendo72 — COMMENTED** [review]
That's not what this commit is about.

This commit was about:
- Added Cata checks same as GuideList
- positioned white text above title bar inside frame window (I believe Spoony is tweaking this in Retail)

File: WoWPro/Libraries/HereBeDragons/CHANGELOG.md
Comment (Cagomei) [inline WoWPro/Libraries/HereBeDragons/CHANGELOG.md:1]: This change needs to be discarded. IDK how this happened.

Comment (Hendo72) [reply WoWPro/Libraries/HereBeDragons/CHANGELOG.md:1]: Which change are you referring to? Is it one line specific or the entire commit?

**Comment by Hendo72** [comment]
I'm closing this PR and starting a new one with  less commits.

**Pull request closed without merging**